packages/amplify_api_rest/android/src/test/kotlin/com/amazonaws/amplify/amplify_api_rest/AmplifyApiRestTest.kt

```diff
@@ -0,0 +1,414 @@
+
+        var data = "{\"error\":\"get call failed!\"}".toByteArray()
+        var restResponse = RestResponse(400, data)
+
+        Mockito.doAnswer { invocation: InvocationOnMock ->
+            (invocation.arguments[1] as Consumer<RestResponse>).accept(
+                    restResponse
+            )
+            mockRestOperation
+        }.`when`(mockApi).get(
+                ArgumentMatchers.any(RestOptions::class.java),
+                ArgumentMatchers.any(),
+                ArgumentMatchers.any())
+
+        var arguments : Map<String, Any> = mapOf(
+        )
+
+        flutterPlugin.onMethodCall(
+                MethodCall("get", arguments),
+                mockResult
+        )
+
+        verify(mockResult, times(1)).error(
+                "AmplifyApiRestException",
+                "someMsg",
+                "errorDetails"
+        )
+
+    }
+
+    // Cancellation of ongoing rest operation succeeds
+    @Test
+    fun test_cancel_get_returns_success(){
+
+        var data = "{\"success\":\"get call succeed!\",\"url\":\"/items\"}".toByteArray()
+        var restResponse = RestResponse(200, data)
+        var code = "someCode"
+
+        Mockito.doAnswer { invocation: InvocationOnMock ->
+            Assert.assertEquals(
+                    RestOptions.builder().addPath("/items").build(),
+                    invocation.arguments[0]
+            )
+            mockRestOperation
+        }.`when`(mockApi).get(
+                ArgumentMatchers.any(RestOptions::class.java),
+                ArgumentMatchers.any(),
+                ArgumentMatchers.any())
+
+        var arguments : Map<String, Any> = mapOf(
+                "restOptions" to mapOf(
+                        "path" to "/items"
+                ),
+                "code" to code
+        )
+
+        flutterPlugin.onMethodCall(
+                MethodCall("get", arguments),
+                mockResult
+        )
+
+        val mockCancelResult: MethodChannel.Result = mock(MethodChannel.Result::class.java)
+
+        flutterPlugin.onMethodCall(
+                MethodCall("cancel", code),
+                mockCancelResult
+        )
+
+        verify(mockResult, times(0)).success(
+                mapOf(
+                        "data" to restResponse.data.rawBytes
+                )
+        )
+
+        verify(mockCancelResult, times(1)).success(
+                "Operation Canceled"
+        )
+    }
+
+    // Cancellation of non existing rest operation fails
+    @Test
+    fun test_cancel_get_returns_error(){
+
+        var data = "{\"success\":\"get call succeed!\",\"url\":\"/items\"}".toByteArray()
+        var restResponse = RestResponse(200, data)
+        var code = "someOldCode"
+
+        val mockCancelResult: MethodChannel.Result = mock(MethodChannel.Result::class.java)
+
+        flutterPlugin.onMethodCall(
+                MethodCall("cancel", code),
+                mockCancelResult
+        )
+
+        verify(mockCancelResult, times(1)).error(
```

Comment:
`times(1)` is not needed. `verify(mockCancelResult)` is short-hand for `times(1)`.

Reply:
thanks!  Removed that from the unit test